subjects/rot14.en.md

```diff
@@ -2,9 +2,7 @@
 
 ### Instructions
 
-Write a function `rot14` that returns the `string` within the parameter transformed into a `rot14 string`.
-
--   There is the need to know what `rot13` stands for.
```

Comment:
I believe that this subject was supposed to not be that clear, you can easly google rot13 and try to figure it out.
Also the broken program links are currently placeholders the links will be replaced, unrelated to the issue.

Reply:
Students cant google for rot13 during the exam. They were asking what rot13 is, cuz they didnt do it before.